pkg/frontend/adminactions/resources_list.go

```diff
@@ -30,6 +30,10 @@ func (a *azureActions) ResourcesList(ctx context.Context) ([]byte, error) {
 	if err != nil {
 		a.log.Warnf("error when getting network resources: %s", err)
 	}
+	armResources, err = a.appendAzureDiskEncryptionSetResources(ctx, armResources)
+	if err != nil {
+		a.log.Warnf("error when getting DiskEncryptionSet resources: %s", err)
+	}
```

Comment:
I might need to update `armResources := make([]arm.Resource, 0, len(resources)+3)` above to avoid allocations on append.

Reply:
Lets discuss the intention here? Why is it `len()+3`? This append function is optional based on existance of the diskEncryptionSet.